src/components/DropList/DropList.ListItem.jsx

```diff
@@ -103,6 +109,17 @@ const ListItem = forwardRef(
   }
 )
 
+/**
+ * If `substr` is passed, we want to emphasize that with html's <strong />
+ */
+export function getListItemText({ label, emphasize, substr }) {
+  const __html = emphasize ? emphasizeSubstring(label, substr) : label
+
+  return {
+    __html,
+  }
+}
+
```

Comment:
Out of curiosity, what's the reason for creating an HTML string and using `dangerouslySetInnerHTML` vs. creating and passing React children to the `ListItemTextUI` component? 

If the answer is "it was faster and easier to write the utils for doing this that way!" that is totally an acceptable answer to me, just curious!

Reply:
no reason, different ways to achieve the same things. I like to live dangerously every now and then